packages/mermaid/src/diagrams/pie/pieRenderer.ts

```diff
@@ -125,12 +127,81 @@ export const draw: DrawDefinition = (text, id, _version, diagObj) => {
     .style('text-anchor', 'middle')
     .attr('class', 'slice');
 
-  group
+  const titleText = db.getDiagramTitle();
+  const maxAvailableWidth = pieWidth - MARGIN;
+  let fontSize = 25;
+  const minFontSize = 8;
```

Comment:
8 looks too small, maybe 10?

```suggestion
  const minFontSize = 10;
```